### ISSUE TITLE
Add support for "ruffle-skip" data tag to automatically skip objects

### ISSUE THREAD

**Issue by ErzenXz** [ruffle-rs/ruffle#10785]
I would like to request a new feature for the Ruffle player that would allow it to automatically skip objects with a "ruffle-skip" data tag. This would be useful for cases where certain objects in a SWF file should not be displayed or processed by Ruffle.

Currently, there is no built-in way to skip objects in this manner, so users must manually edit the SWF file or modify the HTML code to exclude the objects they do not want to display.

By adding support for the "ruffle-skip" data tag, Ruffle could automatically skip any objects in a SWF file that have this tag, making it much easier for users to control which objects are displayed and which are not.

Thank you for your consideration, and please let me know if you have any questions or feedback on this feature request.

**Comment by Lord-McSweeney** [comment]
Currently the only way to do this is to add a `data-ruffle-optout` attribute to the main `<html>` tag:

https://github.com/ruffle-rs/ruffle/blob/292e068e487855d8e157e4c74ceaf582f127f820/web/packages/extension/src/content.ts#L77

**Comment by ErzenXz** [comment]
Yes, i want like specific elements, idk why that even exists because if you dont want it on your entire page just remove the script tag, or maybe can it be done dynamicly? like manipulating the html with js?

**Comment by danielhjacobs** [comment]
> idk why that even exists because if you dont want it on your entire page just remove the script tag, or maybe can it be done dynamicly? like manipulating the html with js?

It exists if your page does not use Ruffle but you want to stop Ruffle extension users from using Ruffle on your page.

I am looking into adding the ability to put `data-ruffle-optout` on any element on an HTML page. That element, and any of its potential children, will not load with Ruffle when Ruffle is in use, but other content on the page will still do so. I'm unsure if that matches your request though. Are you referring to adding an SWF tag for this? We'd rather not add an ability to use tags that were not in Adobe Flash Player, as an anti-goal of any software emulator is distinguishing the emulator usage.

**Comment by kjarosh** [comment]
@ErzenXz could you describe an example use case of this feature and go through it step by step?
